src/main/java/org/commonjava/indy/service/archive/services/LogService.java

```diff
@@ -0,0 +1,276 @@
+package org.commonjava.indy.service.archive.services;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LogService {
+
+    public TraceLog load(String TraceLogJson) {
+        Gson gson = new Gson();
+        return gson.fromJson(TraceLogJson, TraceLog.class);
+    }
+
+    public TraceLog load(InputStreamReader input){
```

Comment:
@yma88 I think maybe we can use these to fit your Track log model, and are we sure to just discard uploads?